media.css

```diff
@@ -0,0 +1,364 @@
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
```

Comment:
- [ ] Please make sure you fix your design it breaks from `768ox` to `1140px` 🙏 

![image](https://user-images.githubusercontent.com/74814780/181595682-becf738f-a74f-4b3a-93fc-b683809a9a81.png)